luminoth/tools/cloud/gcloud.py

```diff
@@ -4,6 +4,7 @@
 import os
 import shutil
 import subprocess
+import sys
```

Comment:
sys and os should be the first imports, right?

Reply:
We don't follow that convention in this repo.